edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/DeviceProxyTest.cs

```diff
@@ -71,16 +74,17 @@ public async Task Cloud2DeviceMessagesForwarded()
                 .Setup(h => h.SendC2DMessageAsync(It.Is<IMessage>(m => m == message), It.Is<IIdentity>(i => i == identity)))
                 .Returns(Task.CompletedTask);
 
-            var sut = new DeviceProxy(identity, twinHandler, m2mHandler, c2dHandler, directMethodHandler);
+            var sut = new DeviceProxy(identity, connectionHanlder, twinHandler, m2mHandler, c2dHandler, directMethodHandler);
 
             await sut.SendC2DMessageAsync(message);
 
-            Mock.Get(directMethodHandler).VerifyAll();
```

Comment:
Why this change?

Reply:
it was a copy-paste error from earlier and it did not call the "verifyall" on the set up mock object. Now it checks the right mock. Earlier the test passed only because nothing was setup on 'directMethodHandler', so verifyAll meant 'verify nothing' - which always passed.